### PR TITLE
Added Recommended Practices

### PR DESCRIPTION
**Pull request by anishasthana**

### CODE REVIEW

File: integration.adoc [hunk 1]
```diff
@@ -15,6 +15,19 @@ so rather than repeat that information here, it's recommended to be familiar
 with the practices in that document. Points of particular interest will be
 included in the rest of this guide.
 
+[[recommended-practices]]
+=== Recommended Practices
+
+There are some recommended practices for your images to ensure full compatibility with Red Hat Products. 
+
+* Containers should not run as root.
```
Comment (anishasthana) [inline integration.adoc]: Unsure if "should" is more fitting or "can"/"must" are. This is for all the following.

Comment (jdob) [inline integration.adoc]: I think "should" is fine here. Something like "must" feels awkward in a section titled "recommended".

File: integration.adoc [hunk 2]
```diff
@@ -15,6 +15,18 @@ so rather than repeat that information here, it's recommended to be familiar
...
+** Containers running as root represent a security vulnerability as a process that breaks out of the container will retain root privileges on the host system.
```
Comment (jdob) [inline integration.adoc]: Can you add a link to the section of the guide that talks about how to do this?

File: integration.adoc [hunk 3]
```diff
@@ -15,6 +15,18 @@ so rather than repeat that information here, it's recommended to be familiar
...
+* Containers should not request host-level privileges.
+** Containers requiring host-level privileges may not function correctly in all environments- namely those that the application deployer does not have full control over the host system.
```
Comment (jdob) [inline integration.adoc]: I think we should add a line here talking about how it's limiting them from running inside of OpenShift Online and Dedicated.

File: integration.adoc [hunk 4]
```diff
@@ -15,6 +15,18 @@ so rather than repeat that information here, it's recommended to be familiar
...
+* Containers should use a Red Hat provided base image. They should also not modify content provided by Red Hat packages or layers.
```
Comment (jdob) [inline integration.adoc]: I know it's in the sample Dockerfile, but can you add in the URL that should be used in the FROM field? The registry.redhat.com one (or whatever it actually is, that's off the top of my head).

**Comment by anishasthana** [comment]
@jdob ptal

**Pull request merged**